manifests/repo/vfabric.pp

```diff
@@ -7,12 +7,12 @@
   $osver = split($::operatingsystemrelease, '[.]')
 
   yum::managed_yumrepo { 'vfabric':
-    descr          => 'vFabric 5.2 Repo - $basesearch',
-    baseurl        => "http://repo.vmware.com/pub/rhel${osver[0]}/vfabric/5.2/\$basearch",
+    descr          => 'vFabric 5.3 Repo - $basesearch',
+    baseurl        => "http://repo.vmware.com/pub/rhel${osver[0]}/vfabric/5.3/\$basearch",
     enabled        => 1,
     gpgcheck       => 1,
     failovermethod => 'priority',
-    gpgkey         => "http://repo.vmware.com/pub/rhel${osver[0]}/vfabric/5.2/RPM-GPG-KEY-VFABRIC-5.2-EL${osver[0]}",
+    gpgkey         => "http://repo.vmware.com/pub/rhel${osver[0]}/vfabric/5.3/RPM-GPG-KEY-VFABRIC-5.3-EL${osver[0]}",
```

Comment:
Maybe it's better to create a new vfabric53 class or, maybe better, expose here a parameter "version" and use it in the code (default value should be 5.2 to keep th same behaviour"